notebook/i18n/fr_FR/LC_MESSAGES/nbjs.po

```diff
@@ -868,11 +869,11 @@ msgstr ""
 
 #: notebook/static/notebook/js/notebook.js:470
 msgid "Restart and Run All Cells"
-msgstr "Relancer et Exécuter Toutes les Cellules"
+msgstr "Relancer et exécuter toutes les cellules"
```

Comment:
I tried to coordinate my efforts with @agatheliguori who removed Title Case entirely in another file on #4074. I'll probably do another pass on this file as well.

Reply:
OK, just to avoid dying idiot or doubting :  the rationale behind those changes is that ortho-typography differs between French and English on how to capitalize titles, right ?

Reply:
Right